libraries/AP_BattMonitor/AP_BattMonitor_DJI.cpp

```diff
@@ -0,0 +1,79 @@
+#include <AP_HAL/AP_HAL.h>
+#include "AP_BattMonitor_DJI.h"
+#include <GCS_MAVLink/GCS.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <AP_Math/crc.h>
+
+/*
+  "battery" monitor for DJI Battery via serial port.
+
+   For future Referance:
+   Total Voltage : 8,9
+   Percentage : 17
+   Cell1 :  20,21
```

Comment:
Does this mean you can be filling in the cell voltages array and provide this data to the GCSlogs as well?

Reply:
individual cell voltage and number of cells data is available already in the pktbuf, need to integrate that.

Reply:
added individual cell monitoring